e2e/teardown.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+
+docker kill iqengine
+docker kill iqenginedb
+docker network rm iqenginenet
+docker container prune --force
```

Comment:
Wouldn't make more sense to move this to a docker compose?

Reply:
outdated